src/source/PeerConnection/PeerConnection.c

```diff
@@ -844,6 +844,12 @@ STATUS setRemoteDescription(PRtcPeerConnection pPeerConnection, PRtcSessionDescr
     CHK_STATUS(setTransceiverPayloadTypes(pKvsPeerConnection->pCodecTable, pKvsPeerConnection->pRtxTable, pKvsPeerConnection->pTransceievers));
     CHK_STATUS(setReceiversSsrc(pSessionDescription, pKvsPeerConnection->pTransceievers));
 
+    if (getenv(DEBUG_LOG_SDP)) {
```

Comment:
Recommend if (NULL != getenv(DEBUG_LOG_SDP)

Also, think whether you can make a single macro that does the ifs and printouts. It should obviously, take two params then - the string denoting whether it's local or remote and the string that's the sdp

Reply:
MACRO is fine. But, I dont see a need for it here. I will let it be an if statement